Cython/Build/Dependencies.py

```diff
@@ -845,7 +860,13 @@ def copy_to_build_dir(filepath, root=root):
         for source in m.sources:
             base, ext = os.path.splitext(source)
             if ext in ('.pyx', '.py'):
-                if m.language == 'c++':
+                if np_pythran:
+                    c_file = base + '.cpp'
+                    options = pythran_options
+                    m.include_dirs.append(pythran_include_dir)
+                    m.extra_compile_args.extend(('-std=c++11','-DENABLE_PYTHON_MODULE','-D__PYTHRAN__'))
```

Comment:
Done in https://github.com/cython/cython/pull/1607/commits/cad11dc11c7fe33ba97936c632b5cf801c9738c6 !